Examples/Tests/radiation_reaction/test_const_B_analytical/inputs

```diff
@@ -0,0 +1,66 @@
+#Automatically generated inputfile
+#Run check.py without arguments to regenerate
+#
+
+max_step = 80
+amr.n_cell = 80 80 80
+amr.max_level = 0
+amr.blocking_factor = 8
+amr.max_grid_size = 8
+geometry.coord_sys   = 0
+geometry.is_periodic = 1 1 1
+geometry.prob_lo = -1e-06 -1e-06 -1e-06
+geometry.prob_hi = 1e-06 1e-06 1e-06
+warpx.do_pml = 0
+algo.charge_deposition = standard
+algo.field_gathering = standard
+warpx.cfl = 1.0
+
+particles.nspecies = 5
+particles.species_names =  ele_para0 ele_perp0 ele_perp1 ele_perp2 pos_perp2
+
+amr.plot_int = 80
+
+warpx.plot_raw_fields = 0
+
+ele_para0.charge = -q_e
+ele_para0.mass = m_e
+ele_para0.injection_style = "SingleParticle"
+ele_para0.single_particle_pos = 0.0 0.0 0.0
+ele_para0.single_particle_vel = 285.7142857142856 428.5714285714285 857.142857142857
+ele_para0.single_particle_weight = 0.0001
+ele_para0.do_classical_radiation_reaction = 1
+
+ele_perp0.charge = -q_e
+ele_perp0.mass = m_e
+ele_perp0.injection_style = "SingleParticle"
+ele_perp0.single_particle_pos = 0.0 0.0 0.0
+ele_perp0.single_particle_vel = -47.91574237499548 6.388765649999412 12.777531299998806
+ele_perp0.single_particle_weight = 0.0001
+ele_perp0.do_classical_radiation_reaction = 1
+
+ele_perp1.charge = -q_e
+ele_perp1.mass = m_e
+ele_perp1.injection_style = "SingleParticle"
+ele_perp1.single_particle_pos = 0.0 0.0 0.0
+ele_perp1.single_particle_vel = 1.4274296030894868e-14 178.88543819998318 -89.44271909999159
+ele_perp1.single_particle_weight = 0.0001
+ele_perp1.do_classical_radiation_reaction = 1
+
+ele_perp2.charge = -q_e
+ele_perp2.mass = m_e
+ele_perp2.injection_style = "SingleParticle"
+ele_perp2.single_particle_pos = 0.0 0.0 0.0
+ele_perp2.single_particle_vel = -958.3148474999095 127.77531299998823 255.55062599997612
+ele_perp2.single_particle_weight = 0.0001
```

Comment:
We don't have test particles in WarpX. It could be a nice option, but for now I think you have the right way to go.

Reply:
Ok! I've further reduced the weight in the latest version.